### ISSUE TITLE
Refund: Additional Fee Refund: Display is incorrect

### ISSUE THREAD

**Issue by Tas4tari** [big-neon/bigneon#209]
The order total was $3.75, however the front-end displays the refund as $2.75.

On further investigation it was noted that the entire amount of $3.75 was refunded.
![Image Pasted at 2019-6-26 18-05](https://user-images.githubusercontent.com/50987039/60197991-dc212780-9840-11e9-879a-bf6d596cf316.png)

**Comment by mikethetike** [comment]
Is it worth fixing this at this moment? This screen looks like the order management screen which will be replaced soon anyway?

**Comment by Jasonvdb** [comment]
Yeah agreed, working on the new order management now. Can leave this open and just close it when it's corrected in the new version.
